corehq/apps/app_manager/models.py

```diff
@@ -4452,18 +4452,15 @@ def get_odk_qr_code(self, with_media=False, build_profile_id=None, download_targ
 
     def generate_shortened_url(self, view_name, build_profile_id=None):
         try:
-            if bitly.BITLY_CONFIGURED:
-                view_url = reverse(view_name, args=[self.domain, self._id])
-                if build_profile_id is not None:
-                    long_url = urljoin(
-                        self.url_base,
-                        f'{view_url}?profile={build_profile_id}'
-                    )
-                else:
-                    long_url = urljoin(self.url_base, view_url)
-                shortened_url = bitly.shorten(long_url)
+            view_url = reverse(view_name, args=[self.domain, self._id])
+            if build_profile_id is not None:
+                long_url = urljoin(
+                    self.url_base,
+                    f'{view_url}?profile={build_profile_id}'
```

Comment:
I know you didn't change this, but is it possible for `view_url` to contain parameters like `?name=value`? If yes, this will create a malformed URL.

Reply:
I don't think `reverse` can return a url with parameters like that, but could be wrong?